core/utils/image_urls.py

```diff
@@ -1,16 +1,30 @@
 from django.conf import settings
-
 from django.utils.http import urlquote
 
+RESIZE = 0
```

Comment:
This is interesting.  I've not thought of assigning something like this before.  Somehow it makes me a little bit nervous, since 0 and 1 aren't terribly descriptive of resize vs tile servers.  I can't think of a nicer way off the top of my head, though...  :/